README.md

```diff
@@ -14,7 +14,8 @@ None.
 
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| freeipa_days_before_inactive | The number of days a user can go without logging in before his or her account is determined to be inactive and is disabled. | `45` | No |
+| freeipa_days_before_inactive | The number of days an account can go without being authenticated before it is determined to be inactive and is disabled. | `45` | No |
+| freeipa_disable_inactive_users | Whether or not to set up FreeIPA and create a systemd timer and service to disable inactive users. | `false` | No |
```

Comment:
Minor complaint but I find the phrasing a bit ambiguous around the "set up FreeIPA" part. It would be nice to tie it more directly tie it to the second half of the sentence by providing some mention what "set up FreeIPA" entails.

Reply:
Please see commit 494db64.